src/network/udp.py

```diff
@@ -36,11 +42,7 @@ def __init__(self, host=None, sock=None, announcing=False):
             self.set_socket_reuse()
             logger.info("Binding UDP socket to %s:%i", host, UDPSocket.port)
             self.socket.bind((host, UDPSocket.port))
-            #BINDTODEVICE is only available on linux and requires root
-            #try:
-                #print "binding to %s" % (host)
-                #self.socket.setsockopt(socket.SOL_SOCKET, socket.SO_BINDTODEVICE, host)
-            #except AttributeError:
+            # BINDTODEVICE is only available on linux and requires root
```

Comment:
Can be removed. I wanted to have the option to specify a network interface rather than an IP address when binding, but as the comment says it only works on linux and as root, so it's useless and can be dropped.